tests/job/test_jobs.py

```diff
@@ -90,3 +90,8 @@ def test_succeeded(self):
         self.check_column_names(resp, expect_col_names)
         expect_values = [[0]]
         self.check_result(resp, expect_values)
+
+    def test_balance_job(self):
+        resp = self.client.execute('CREATE SPACE space_for_balance(partition_num=9, replica_factor=1);'
+                                   'USE space_for_balance;')
+        self.check_resp_succeeded(resp)
```

Comment:
Do you forget to test the balance leader job?

Reply:
yeah still in processing